### PR TITLE
[draft] Contribution Guide updates

### PR DESCRIPTION
**Pull request by LadyBluenotes**
Updating the contribution guide.

Addressing things as noted in issue #279

### CODE REVIEW

**Comment by LadyBluenotes** [comment]
@michaelessiet can you review content to make sure I haven't missed anything?

@jccpalmer could you please edit this?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,103 +1,148 @@
...
-> **Note** Existing PRs and Issues need reviewing, triaging, and feedback, too! You can make valuable contributions by commenting, suggesting, testing, researching, brainstorming and generally helping in all areas on GitHub!
+#### Examples of helpful PRs
+- PRs addressing an existing fix
+- unsolicited PRs addressing typos, broken links, and other minor problems
 
-## Navigating the Codebase
+### Edit this page via GitHub
+Every page on [﻿docs.solidjs.com ](https://docs.solidjs.com/)﻿has an **Edit this page** link at the bottom. You can click on that button to edit the source code for that page in **GitHub**.
 
-Our site is build on [SolidStart](https://github.com/solidjs/solid-start). This is a framework that's still in development, so if you have any issues, check out the #docs or #solid-start channels on Discord. (In fact, the SolidStart team is working on documentation [within our repo](https://github.com/solidjs/solid-docs-next/tree/main/src/routes/start), too!)
```
Comment (michaelessiet) [inline CONTRIBUTING.md:68]: built*

**Review by michaelessiet — APPROVED** [review]
Looks good

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,103 +1,148 @@
-_Read the [Solid Docs Manifesto](https://github.com/solidjs/solid-docs-next#the-solid-docs-manifesto) first!_
+# CONTRIBUTION GUIDE
 
-# Contributing to the Solid Docs
+Thank you for helping us make this project great and being a part of the Solid community!
 
-We welcome contributions of any size and contributors of any skill level. As an open source project, we believe in giving back to our contributors. We are happy to help with guidance on PRs, technical writing, and turning a feature idea into a reality.
+We welcome contributions from anyone, regardless of your skill level of the size of the contribution. We are happy to help with guidance on PRs, technical writing, and turning features into realities.
```
Comment (jccpalmer) [inline CONTRIBUTING.md:5]:
```suggestion
We welcome contributions from anyone, regardless of your skill level.
We are happy to help with guidance on PRs, technical writing, and turning features into realities.
```

**Review by jccpalmer — COMMENTED** [review]
Quick edit, sorry for the delay.

**Review by LadyBluenotes — COMMENTED** [review]
Thank you!

**Pull request merged**